controls/autocomplete/migrate-from-xamarin.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: Migrating from Xamarin
 page_title: Migrating the AutoComplete from Xamarin.Forms to .NET MAUI
-description: "Learn how to migrate the Telerik UI for Xamarin AutoComplete to the Telerik UI for .NET MAUI framework by updating the namespaces and the incompatible NuGet packages."
+description: Learn how to migrate the Telerik UI for Xamarin AutoComplete to the Telerik UI for .NET MAUI framework by updating the namespaces and the incompatible NuGet packages
```

Comment:
Use closing punctuation in the meta descriptions and insert a period at the end of the sentence.
```suggestion
description: Learn how to migrate the Telerik UI for Xamarin AutoComplete to the Telerik UI for .NET MAUI framework by updating the namespaces and the incompatible NuGet packages.
```